app/models/user.rb

```diff
@@ -482,6 +485,42 @@ def title
     name
   end
 
+  def number_visits_per_day
+    return user.sign_in_count.round(1) if user.created_at.today?
```

Comment:
if the `user` was created on the same day, we return the number of times the `user` signed in (to avoid dividing by zero)